### PR TITLE
add more infos for using lab on windows wsl

### PR DESCRIPTION
**Pull request by sybnex**
Added some missing infos for better usage in windows systems

### CODE REVIEW

File: content/en/docs/06/_index.md [hunk 1]
```diff
@@ -103,6 +103,14 @@ Once you have the IP you can connect with the command `mysql -h172.17.0.2 -uroot
 
 {{% alert title="Note" color="primary" %}}
 The mysql client must be installed on your computer. On Windows, you can use the binary from the ZIP archive at <https://dev.mysql.com/downloads/mysql/>.
+
+Also on Windows you must use port forwarding on the docker container to access the Database:
```
Comment (bliemli) [inline content/en/docs/06/_index.md:107]:
```suggestion
Also on Windows you must use port-forwarding to access the database:
```

File: content/en/docs/11/_index.md [hunk 2]
```diff
@@ -41,6 +41,16 @@ RUN apt-get update && \
...
+{{% alert title="Note" color="primary" %}}
+Copy the content above. Switch to the vim editor and use the following commands to insert and save everything:
+
+* "i"
+* "crtl-v"
+* "esc"
+* ":wq"
+* "enter"
+{{% /alert %}}
+
```
Comment (bliemli) [inline content/en/docs/11/_index.md:53]: We already tell participants to use their editor of choice (line 35), so explaining how to use `vim` seems a bit off here.

**Review by bliemli — CHANGES_REQUESTED** [review]

File: content/en/docs/11/_index.md (hunk 2 above)
Comment (sybnex) [reply content/en/docs/11/_index.md:53]: I was just wondering finding the "vim Dockerfile" command. Instead we should write then "open a editor of your choice"???

Comment (bliemli) [reply content/en/docs/11/_index.md:53]: That's true, that's misleading too. Maybe we should just remove the command and merge lines 29 and 35 saying something like:

```
Add the following content to the Dockerfile using your editor of choice:
```

**Comment by sybnex** [comment]
@bliemli could you have another look?

**Review by bliemli — APPROVED** [review]

**Pull request merged**